web/modules/custom/dpl_admin/assets/dpl_admin.css

```diff
@@ -184,3 +184,12 @@
 body:has(.eventseries-form--confirm) .form-actions {
   display: none;
 }
+
+/**
+  A very simple element, used in Drupal forms when displaying warnings.
+ */
+.dpl-form-warning {
+  color: #d80404;
+  font-weight: bold;
+  max-width: 500px;
+}
```

Comment:
Consider using [something that relates to the Gin admin theme](https://git.drupalcode.org/project/gin/-/blob/8.x-3.x/styles/base/_form.scss?ref_type=heads#L406).

This could make the addition appear more in tune with the remaining admin interface.

Reply:
I did try that class, but, I prefer my own styling :)